x-pack/plugins/global_search_bar/public/components/search_bar.tsx

```diff
@@ -249,7 +250,11 @@ export const SearchBar: FC<SearchBarProps> = (opts) => {
         console.log('Error trying to track searchbar metrics', err);
       }
 
-      navigateToUrl(url);
+      if (event.ctrlKey || event.metaKey) {
+        window.open(url, '_blank');
+      } else {
+        navigateToUrl(url);
+      }
```

Comment:
Personally, I'm a big fan of holding the Shift key and clicking links to open them in a new window (I prefer to have too many windows vs too many tabs, I guess). The Spaces selector does this, so it's just a matter of using a little bit more of [the example code](https://github.com/elastic/kibana/blob/main/x-pack/plugins/spaces/public/nav_control/components/spaces_menu.tsx#L164-L170). I realize that's not covered verbatim in the issue you're fixing, but since you're working in this code, would you mind making an update to handle these cases?

```typescript
      if (event.shiftKey) {
        // open in new window
        window.open(url);
      } else if (event.ctrlKey || event.metaKey) {
        // open in new tab
        window.open(url, '_blank');
      } else {
        navigateToUrl(url);
      }
```

The code in Spaces also handles "middle click" to open in a new tab, but I don't think that code is actually working, so I wouldn't worry about it.